src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ...;
```